README.md

```diff
@@ -60,6 +60,10 @@ To build the documentation web pages, simply run
 
     $ make html
 
+Please note that you will need to have [nodejs](https://nodejs.org)
+installed to produce HTML content with the above command, in particular
+a `node` executable must to be in your `PATH`.
```

Comment:
"should be" sounds better. Thanks for the changes.

Reply:
Fixed.

Please note I removed also a reference to panda (different commit, so cherry-picking is available).